### PR TITLE
Ready: Rename everything "CorrugatedIron" to "RiakClient"

### PR DESCRIPTION
**Pull request by alexmoore**
99% of the `CorrugatedIron`'s have been replaced with `RiakClient` or `riak-dotnet-client` depending on if it's a github reference or not.  Any remaining `CorrugatedIron`'s are there for historical purposes, or have an external component that has yet to be created (like an official nuget release).

### CODE REVIEW

**Comment by alexmoore** [comment]
@lukebakken Ready

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-CorrugatedIron - a .NET client for Riak
+RiakClient - a .NET client for Riak
```
Comment (lukebakken) [inline README.md]: `RiakClient` -> `Riak .NET Client` - a client library for Riak for applications running on the .NET CLR.

Does the following seem OK?

* "Riak .NET Client" - the official name of the client when spaces and using the string ".NET" is OK, like in documentation or in the soon-to-be-created (or renamed) NuGet package title.
* `riak-dotnet-client` - the GH repo name and name of the project when it has to be part of a URL.
* `RiakClient` - the NuGet package name, and `.dll` name. Using [this package](http://www.nuget.org/packages/Microsoft.Net.Http/) as an example, the title is "Riak .NET Client" and installation command is `Install-Package RiakClient`. The other option is to follow the [Ruby gem](https://rubygems.org/gems/riak-client) scheme in which case the dll would be `riak-dotnet-client.dll` and NuGet package command `Install-Package riak-dotnet-client`

Comment (alexmoore) [inline README.md]: Yes.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Please see the [Release Notes](RELNOTES.md) document.
 License
 -------
 
-**CorrugatedIron** is Open Source software released under the Apache 2.0 License. Please see the [LICENSE](LICENSE) file for full license details.
+**RiakClient** is Open Source software released under the Apache 2.0 License. Please see the [LICENSE](LICENSE) file for full license details.
```
Comment (lukebakken) [inline README.md]: "Riak .NET Client" etc etc etc

**Comment by lukebakken** [comment]
:shipit: anything missed can be caught later. Tests all pass!

**Pull request merged**